kotlin/wmts-layer/README.md

```diff
@@ -1,34 +1,42 @@
-# WMTS Layer
+# WMTS layer
 
-This sample created a WMTS layer from the given url and the first layer id, which is retrieved by loading the associated WMTS service and obtaining the layer info list. Then the WMTS layer will be added to the map as a basemap layer.
+Display a layer from a Web Map Tile Service.
 
-![WMTS Layer App](wmts-layer.png)
+![Image of WMTS layer](wmts-layer.png)
 
-## Features
+## Use case
+
+WMTS services can have several layers. You can use Runtime to explore the layers available from a service. This would commonly be used to enable a browsing experience where users can choose which layers they want to display at run time.
+
+## How to use the sample
+
+The layer will be displayed automatically. Use the buttons to choose a different method of loading the layer.
```

Comment:
It looks like there are no buttons on this sample, so you can probably cut the second sentence.